data/importer.py

```diff
@@ -0,0 +1,49 @@
+from models import (
+    Category,
+    Criterion,
+    State,
+)
+
+from services import (
+    update_or_create_category,
+    update_or_create_criterion,
+    update_state
+)
+
+
+def import_categories(categories_json):
```

Comment:
This class implements the `import_categories` and `import_states` methods, which perform the actual data imports based on the json queried from the CMS.